### DISCUSSION TITLE
need syntax for executing action on specific day

### DISCUSSION THREAD

**Discussion by jarod184** [arendst/Tasmota#16122] [Q&A] [answered]
hi,
i read the help but didn't find what i needed.
i would like to inhibate a button every friday at 16:00 and reactivate it every saturday at 20:00 
to inhibate a button, it is quite simple i use setoption 65
but i missing the syntax for the schedule part

could anyone help me please ?

**Comment by sfromis** [comment]
Day-specific timers are on the web UI page Configuration / Timers, and you can have rules to react to the timers.

**Comment by jarod184** [comment]
Yes i’ve seen that but I don’t know how to llink the tasmota timer with rule<n> ☹

**Reply by sfromis** [reply]
Did you read the rules docs?
https://tasmota.github.io/docs/Rules/#execute-several-commands-when-a-timer-expires

**Comment by pkkrusty** [comment, marked as answer]
As sfromis suggested, the perfect example is in the docs. Set up a timer to trigger Rule on Friday at 16:00 and one for Saturday at 20:00, then in console type `rule1 On clock#timer=1 do setoption65 1 endon on clock#timer=2 do setoption65 0 endon`

But maybe switchmode 15 would be better?

**Comment by jarod184** [comment]
Unfortunately, « setoption65 1 » doesn’t deactivate the buttons as written in the docs. 

Maybe I’m missing something because even in the console, typing directly setoption65 1 doesn’t do nothing ☹

**Reply by barbudor** [reply]
Because setoption65 is not correct
It is [setoption73](https://tasmota.github.io/docs/Commands/#setoptions)

**Comment by jarod184** [comment]
Now it works 😊

Thx !!!!
